src/Resources/config/twig.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sonata.timeline.twig.intl_extension" class="Twig\Extensions\IntlExtension">
```

Comment:
This is needed?

Reply:
When I've used them in a project, I had to add them as services. In [the docs](https://twig.symfony.com/doc/3.x/filters/format_date.html) it says:
```php
use Twig\Extra\Intl\IntlExtension;

$twig = new \Twig\Environment(...);
$twig->addExtension(new IntlExtension());
```
which I haven't check but I guess is what `twig.extension` tag does.

Reply:
Okay then. We could keep this for the stable release to stay BC. We should remove this service and update the installation instruction for the next major release.